client/my-sites/site-settings/section-import.jsx

```diff
@@ -89,6 +91,15 @@ class SiteSettingsImport extends Component {
 		this.updateFromAPI();
 	}
 
+	componentDidUpdate() {
+		const { engine, site } = this.props;
+		const { importers: imports } = this.state;
+
+		if ( isEmpty( imports ) && 'wix' === engine && site && site.ID ) {
```

Comment:
Are we relying on `isEmpty( imports )` to be `false` after the first time the component updates? Is it possible we trigger this more than once?
If so we could `_.once` the call to `startImport`.

Reply:
Great point -- done!